ignite/src/test/resources/hibernate.properties

```diff
@@ -0,0 +1,11 @@
+#
+# Hibernate OGM, Domain model persistence for NoSQL datastores
+#
+# License: GNU Lesser General Public License (LGPL), version 2.1 or later
+# See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+#
+
+hibernate.search.lucene_version=LUCENE_CURRENT
+hibernate.ogm.datastore.provider=org.hibernate.ogm.datastore.ignite.impl.IgniteDatastoreProvider
```

Comment:
I would set this to IGNITE_EXPERIMENTAL, this way if you change the path or the name of the class you don't need to touch this file. It also make sure that the constant is correct